.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx commitlint -e 
```

Comment:
This is only checking local commits via the local git hooks installed by https://www.npmjs.com/package/husky but it could also be setup to run in the pr-check workflow: https://commitlint.js.org/#/guides-ci-setup.

Reply:
wow, this is pretty cool! I think using it in CI could be powerful, willing to wait for another commit / issue for that :)

Reply:
Sure, I added another commit for that.